apps/test-suite/TestModules.js

```diff
@@ -175,5 +175,5 @@ export function getTestModules() {
     modules.push(optionalRequire(() => require('./tests/Cellular')));
     modules.push(optionalRequire(() => require('./tests/BarCodeScanner')));
   }
-  return modules.filter(Boolean).sort((a, b) => a.name.toLowerCase() > b.name.toLowerCase());
+  return modules.filter(Boolean).sort((a, b) => a.name.toLowerCase() > b.name.toLowerCase() ? 1 : -1);
```

Comment:
Can we use [`localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) instead?

Reply:
it requires [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl). from my test, at least for jsc without Intl, the case insensitive comparison is not supported. 

code like this:
`a.name.localeCompare(b.name, 'en', { sensitivity: 'base' });`

Reply:
> it requires [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl). from my test, at least for jsc without Intl, the case insensitive comparison is not supported.

Oh, TIL, I didn't know that calling it without the locale still requires `Intl`, thank you for the explanation! :)